.github/workflows/e2e-tests-ci.yml

```diff
@@ -10,22 +10,33 @@ jobs:
     strategy:
       matrix:
         node-version: [14.x, 16.x]
+    env:
+      # Whichever Docker containers are needed to pass e2e tests, define them here
+      DOCKER_CONTAINERS: 'sso feed-discovery auth login kong rest meta'
```

Comment:
I am just wondering if we want to add and test Parser e2e tests, if this means we'll have to add more images into this line here?

Reply:
Correct.